### PR TITLE
Feat/59 setup backend project structure

### PR DESCRIPTION
**Pull request by wynonagcc**
Please use this format and structure when implementing the backend. I've been looking into and trying ways to avoid initializing the sequelize instance every time we want to connect to db. And the most effective way that I could find is by using sequelize models, but I still can't get it to work correctly. These are just some TODOs that we can look into next, let's prioritize implementing our API endpoints for now (i.e., follow what we're doing for the `testEndpoint`). You can test the sample endpoint by opening `localhost:8000/appstatus` or `localhost:8000/user/testEndpoint` on your browser after doing all the steps mentioned in the README.

### CODE REVIEW

File: fujiji-server/src/repositories/user.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const sequelize = new Sequelize(dbConfig.NAME, dbConfig.USERNAME, dbConfig.PASSWORD, {
+  host: dbConfig.HOST,
+  dialect: 'mssql',
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 10000,
+  },
+});
```
Comment (devin-efendy) [inline fujiji-server/src/repositories/user.js:12]: We probably can extract this out and put it in separate module like `db.js` maybe (?)

and do something like this: `module.exports = sequelize` so we can re-use this by using `require`

Then, at the other file that we want to use we can just use this:
```
const sequelize = require('/path/to/module')
```

Comment (wynonagcc) [reply fujiji-server/src/repositories/user.js:12]: I had tried that, but still didn't work correctly

Comment (devin-efendy) [reply fujiji-server/src/repositories/user.js]: np at all! lol I was so confused to with module.exports the first time I use node

everything lgtm now!

**Review by devin-efendy — APPROVED** [review]

**Comment by devin-efendy** [comment]
Oh just realized you are using `cross-env`. Was this necessary?

**Pull request merged**